microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,149 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1462.37
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1418.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1462.37)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 108.0.1462.37 or higher.
+
+### General
+
+This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1549-prerelease. See **Bug fixes** in the following section.
+
+
+<!-- ====================================================================== -->
+## 1.0.1549-prerelease
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1466-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1549-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 110.0.1549.0 or higher.
+
+### General
+
+#### Experimental features 
+
+*  Added support for the Locale Region API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+##### [Win32/C++](#tab/win32cpp)
+
+*  [ICoreWebView2EnvironmentOptions3 interface]()
+   * [ICoreWebView2EnvironmentOptions3::IsCustomCrashReportingEnabled property(get](), [put)]()
+*  [ICoreWebView2Environment11::get_FailureReportFolderPath property]()
+
+---
+
+*  Added support for the tracking prevention API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+##### [Win32/C++](#tab/win32cpp)
+
+*  [ICoreWebView2ExperimentalEnvironmentOptions2 interface]()
+   * [ICoreWebView2ExperimentalEnvironmentOptions2::EnableTrackingPrevention property(get](), [put)]()
+*  [ICoreWebView2ExperimentalProfile5 interface]()
+   * [ICoreWebView2ExperimentalProfile5::PreferredTrackingPreventionLevel property(get](), [put)]()
+        
+---
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:
+
+*  Added support for the Print API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.PrintAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync?view=webview2-dotnet-1.0.1414-prerelease&preserve-view=true)

```

Comment:
Global in this h2 section: in API Ref URLs, maybe change from:
1.0.1414-prerelease
to:
1.0.1414-prerelease
after API Ref is published.

Reply:
fixed